docs/GettingStartedDocs/README.md

```diff
@@ -33,18 +33,12 @@ the hardware types below.
 
 ### Intel SGX
 
-If you would like to start developing apps with the preview Open Enclave SDK
-release, start here for instructions to install and use the SDK package:
-
-- [Ubuntu 18.04 with SGX hardware](install_oe_sdk-Ubuntu_18.04.md)
```

Comment:
Why this is removed?

Reply:
@radhikaj suggested that this be removed since these documentations are shipped bundled with the releases.

Reply:
@RRathna, I think this can stay. Since it relates to 1804.

Reply:
Adding back steps to install OE SDK.